openedx/core/djangoapps/content_libraries/api.py

```diff
@@ -1001,12 +1002,16 @@ def get_bundle_links(library_key):
             opaque_key = libraries_linked[link_data.bundle_uuid].library_key
         except KeyError:
             opaque_key = None
-        # Append the link information:
+        try:
+            latest_version = blockstore_cache.get_bundle_version_number(link_data.bundle_uuid)
+        except BundleNotFound:
+            # if the bundle doesn't exist, it's a broken link, so ignore it
```

Comment:
Should log an error even if it ignores it -- so someone can track these issues.

Reply:
Oh I haven't addressed this yet sorry.

Reply:
Yep, makes sense.